back/db/migrate/20220822140950_add_fields_and_rename_field_in_static_pages.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class AddFieldsAndRenameFieldInStaticPages < ActiveRecord::Migration[6.1]
+  # rubocop:disable Rails/ApplicationRecord
+  class StubStaticPage < ActiveRecord::Base
+    self.table_name = 'static_pages'
+  end
+  # rubocop:enable Rails/ApplicationRecord
+
+  def change
+    rename_column :static_pages, :body_multiloc, :top_info_section_multiloc
+    change_column_null :static_pages, :top_info_section_multiloc, false
```

Comment:
(should be save because there is no page with `null` body ([Metabase query](https://metabase.hq.citizenlab.co/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7InR5cGUiOiJuYXRpdmUiLCJuYXRpdmUiOnsicXVlcnkiOiJTRUxFQ1QgKlxuRlJPTSBcInByb2R1Y3Rpb25fY2wyX2JhY2tfcmF3XCIuXCJjbDJfcmF3X3N0YXRpY19wYWdlc1wiXG5XSEVSRSAoXCJwcm9kdWN0aW9uX2NsMl9iYWNrX3Jhd1wiLlwiY2wyX3Jhd19zdGF0aWNfcGFnZXNcIi5cImJvZHlfbXVsdGlsb2NcIiBJUyBOVUxMKSIsInRlbXBsYXRlLXRhZ3MiOnt9fSwiZGF0YWJhc2UiOjN9LCJkaXNwbGF5IjoidGFibGUiLCJ2aXN1YWxpemF0aW9uX3NldHRpbmdzIjp7fX0=)))